### PR TITLE
UX changes and extra logs

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

File: custom-recipes/api-connect/recipe.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
         {
             "name": "endpoint_url",
             "label": "URL template",
-            "description": "https://{{subdomain}}.service.com/user/{{username}}/details",
+            "description": "https://{{my_variable}}.service.com/user/{{username}}/details",
```
Comment (MayeulRousselet) [inline custom-recipes/api-connect/recipe.json:48]: {{subdomain|my_variable}}

File: python-connectors/api-connect_dataset/connector.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
         {
             "name": "endpoint_url",
             "label": "URL template",
-            "description": "https://{{subdomain}}.service.com/user/{{username}}/details",
+            "description": "https://{{my_variable}}.service.com/user/{{username}}/details",
```
Comment (MayeulRousselet) [inline python-connectors/api-connect_dataset/connector.json:35]: {{subdomain|my_variable}}

File: python-lib/rest_api_recipe_session.py [hunk 3]
```diff
@@ -52,6 +52,7 @@ def process_dataframe(self, input_parameters_dataframe, is_raw_output):
     def retrieve_next_page(self, is_raw_output):
         page_rows = []
         base_row = copy.deepcopy(self.initial_parameter_columns)
+        logger.info("retrieve_next_page: Calling next page")
```
Comment (MayeulRousselet) [inline python-lib/rest_api_recipe_session.py:55]: maybe log the token?

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]

File: custom-recipes/api-connect/recipe.json (hunk 1 above)
Comment (alexbourret) [reply custom-recipes/api-connect/recipe.json:48]: `subdomain_variable` ? I'm just afraid than `|` will send the wrong message...

File: python-lib/rest_api_recipe_session.py (hunk 3 above)
Comment (alexbourret) [reply python-lib/rest_api_recipe_session.py:55]: :+1 Added logs in has_next_page where the logic takes place.

File: custom-recipes/api-connect/recipe.json (hunk 1 above)
Comment (MayeulRousselet) [reply custom-recipes/api-connect/recipe.json]: https://{{subdomain}}.service.com/user/{{username}}/details or https://{{subdomain}}.service.com/user/{{my_variable}}/details

File: custom-recipes/api-connect/recipe.json [hunk 4]
```diff
@@ -45,7 +45,7 @@
         {
             "name": "endpoint_url",
             "label": "URL template",
-            "description": "https://{{my_variable}}.service.com/user/{{username}}/details",
+            "description": "https://{{subdomain_variable}}.service.com/user/{{username}}/details",
```
Comment (MayeulRousselet) [inline custom-recipes/api-connect/recipe.json:48]: apres reflexion, je mettrais:
https://{{subdomain}}.service.com/user/{{username}}/details
or 
https://{{variable}}.service.com/user/{{username}}/details

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]

**Pull request merged**